src/main/scala/devices/tilelink/CLINT.scala

```diff
@@ -94,6 +94,9 @@ class CLINT(params: CLINTParams, beatBytes: Int)(implicit p: Parameters) extends
 
 /** Trait that will connect a CLINT to a subsystem */
 trait HasPeripheryCLINT { this: BaseSubsystem =>
```

Comment:
should the trait be renamed to "CanHave..." ? @hcook mentioned the naming convention here

Reply:
Yes, that would be my proposal. @solomatnikov suggested something like `CouldGenerate...` which might be even clearer and help to distinguish the `LazyModule` vs `ModuleImp` distinction?

Reply:
I'll change to `CanHave...` to conform with existing pattern.

`CouldGenerate...` can be done in a separate PR.